templates-and-layout/advanced-placeholder.html

```diff
@@ -237,24 +237,30 @@ <h1>Document Title</h1>
                             // In this demo we want to start an empty document with a title.
                             // This does not force having a title for a document, it's simply
                             // a convenience feature.
-                            editor.setContent('<h1><br></h1>');
+                            editor.undoManager.transact(() => {
+                                editor.setContent('<h1><br></h1>');
+                            });
                         }
                     }
-
-                    // Unbind the event listener to avoid the above code running on
-                    // every click inside TinyMCE once the placeholder is removed.
-                    editor.off('click', placeholderManager);
```

Comment:
@lnewson You removed this, just wanted to understand why, as millie told me to include it in the first place.

Reply:
It's because of the `editor.once` right?

Reply:
Yeah, since we use once we don't need to unbind the events.